src/tgraph_bot/graphs/graph_modules/data/data_fetcher.py

```
@@ -11,6 +11,7 @@
 import datetime
 import hashlib
 import logging
+from datetime import datetime, timedelta
```

Ruff (0.12.2)

Redefinition of unused `datetime` from line 11

(F811)